cmd/recover/main.go

```diff
@@ -21,15 +21,13 @@ import (
 	"github.com/athanorlabs/atomic-swap/protocol/xmrtaker"
 	recovery "github.com/athanorlabs/atomic-swap/recover"
 	"github.com/athanorlabs/atomic-swap/swapfactory"
-
-	logging "github.com/ipfs/go-log"
 )
 
 const (
 	flagEnv                  = "env"
 	flagMoneroWalletEndpoint = "monero-endpoint"
 	flagEthereumEndpoint     = "ethereum-endpoint"
-	flagEthereumPrivateKey   = "ethereum-privkey"
+	flagEthereumPrivKey      = "ethereum-privkey"
```

Comment:
Updated this to match the corresponding flag in `swapd`.

Reply:
it would be good to move all the flags to their own package or something so we don't need to redefine them, as many of them are re-used over the various binaries. no need to do that here, but something to keep in mind